code/game/machinery/shieldgen.dm

```diff
@@ -500,6 +500,9 @@
 	src.gen_secondary = B
 	if(A && B)
 		needs_power = 1
+	for(var/mob/living/L in get_turf(src.loc))
+		visible_message("<span class='danger'>The [src] is suddenly occupying the same space as [L]'s organs!</span>")
+		L.adjustBruteLoss(1000)

```

Comment:
I didn't want to deal with the "REMOVING FROM THE ROUND" police when science starts dragging their kills into the field gen.

If this is maintainer permission though I'll go for it.